source/patterns/@aws-solutions-constructs/aws-route53-apigateway/lib/index.ts

```diff
@@ -107,26 +112,22 @@ export class Route53ToApiGateway extends Construct {
         throw new Error('Public APIs require an existingHostedZone be passed in the Props object.');
       } else {
         if (!props.privateHostedZoneProps) {
-          throw new Error('Must supply privateHostedZoneProps to create a private API');
+          throw new Error('Must supply privateHostedZoneProps to create a private API.');
         }
         if (props.privateHostedZoneProps.vpc) {
-          throw new Error('All VPC specs must be provided at the Construct level in Route53ToApiGatewayProps');
+          throw new Error('All VPC specs must be provided at the Construct level in Route53ToApiGatewayProps.');
+        }
+        if (!props.privateHostedZoneProps.zoneName) {
+          throw new Error('Must supply zoneName for Private Hosted Zone Props.');
+        }
+        if ( !this.vpc ) {
+          throw new Error('Must supply an existing VPC for Private Hosted Zone Props.');
```

Comment:
Make this clearer that it's the attribute in the construct props:

Must specify an existingVPC for the Private Hosted Zone in the construct props.